components/dot_menu/dot_menu.tsx

```diff
@@ -326,11 +326,6 @@ export class DotMenuClass extends React.PureComponent<Props, State> {
             return;
         }
 
-        if (this.keysHeldDown.includes(e.key)) {
-            return;
-        }
-        this.keysHeldDown.push(e.key);
-
```

Comment:
I know we originally had this here to avoid running keyboard action more than once if the given action key is held down for a bit rather than quickly tapped.

I'm unable to reproduce that issue on the cloud server for this PR. Was this issue fixed some other way? It seems the menu disappears when you choose an action, and I don't recall if that's how it worked when we added this block here.

Reply:
> Now, the menu closes after an action is taken, be it a click or shortcut.

Make sense to me. Do you happen to know exactly what changed with the code that fixed this issue?

Also, since `this.keysHeldDown` is no longer used, in the sense that it is always empty, we should remove every reference to it in the file.

Reply:
@mickmister, also removed the `onShortcutKeyUp` function.